### ISSUE TITLE
Malicious sites (not present on list)

### ISSUE THREAD

**Issue by etpjhwpknwk** [stamparm/maltrail#14480]
```
https://malwat.ch/malware
https://malwat.ch/msedge.html
https://malwat.ch
```

**Comment by stamparm** [comment]
Any network IOCs for that malware?

**Comment by etpjhwpknwk** [comment]
Other than the URLs themselves, there are no other IOCs I know of at the moment, but that URLs themselves do seem to be malicious or link to other malicious content, and in specific the msedge.html page also seems to contain something that causes computers that run Windows 10 and access it through the Microsoft Edge browser to crash, definitely malicious behavior.

**Comment by MikhailKasimov** [comment]
Hello, @etpjhwpknwk !

```malwat.ch``` site is not malicious itself, dispite even on having such weird name. 

I think, we can put just ```malwat.ch/malware``` path-trail in ```/suspicious/bad_service.txt``` to cover just its "specific" section, where malicious samples could be downloaded from.

Does this sound OK?

Thank you!

**Comment by etpjhwpknwk** [comment]
Yes, that sounds alright, what about the page containing msedge.html though?

**Comment by MikhailKasimov** [comment]
> Yes, that sounds alright, what about the page containing msedge.html though?

![Screenshot_584](https://user-images.githubusercontent.com/7167300/107557437-489d9580-6be2-11eb-9288-43ba92da6a3a.png)
![Screenshot_583](https://user-images.githubusercontent.com/7167300/107557439-49362c00-6be2-11eb-8c33-a2bb462287f6.png)

That what I have in Chrome and Edge on my side... How does this link works on your side?

**Comment by etpjhwpknwk** [comment]
I think it only works on the non-chromium-based version of edge. I would test it but I don't feel like crashing my own computer on purpose at the moment.

**Comment by MikhailKasimov** [comment]
@etpjhwpknwk Anyway, regardless on does crash work for Edge or not, please, meet: https://github.com/stamparm/maltrail/pull/14481/files

**Comment by etpjhwpknwk** [comment]
Alright, that seems fine.

**Comment by stamparm** [comment]
Merged #14481
